src/main/scala/org/broadinstitute/dsde/firecloud/webservice/StaticNotebooksApiService.scala

```diff
@@ -1,31 +1,39 @@
 package org.broadinstitute.dsde.firecloud.webservice
 
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
 import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.dataaccess.DsdeHttpDAO
 import org.broadinstitute.dsde.firecloud.service._
-import org.broadinstitute.dsde.firecloud.utils.StandardUserInfoDirectives
-import spray.routing._
-import spray.http.MediaTypes.`text/html`
+import org.broadinstitute.dsde.firecloud.utils.{HttpClientUtilsStandard, StandardUserInfoDirectives}
 
-trait StaticNotebooksApiService extends HttpService
-  with FireCloudDirectives
-  with StandardUserInfoDirectives {
+import scala.concurrent.ExecutionContext
 
-  private implicit val executionContext = actorRefFactory.dispatcher
+trait StaticNotebooksApiService extends FireCloudDirectives with StandardUserInfoDirectives with DsdeHttpDAO {
+
+  override val http = Http(system)
+  override val httpClientUtils = HttpClientUtilsStandard()
+
+  implicit val executionContext: ExecutionContext
 
   val calhounStaticNotebooksRoot: String = FireCloudConfig.StaticNotebooks.baseUrl
   val calhounStaticNotebooksURL: String = s"$calhounStaticNotebooksRoot/api/convert"
 
   val staticNotebooksRoutes: Route = {
     path("staticNotebooks" / "convert") {
-      requireUserInfo() { _ =>
+      requireUserInfo() { userInfo =>
         post {
-          respondWithMediaType(`text/html`) {
-            requestContext =>
-              // call Calhoun and pass its response back to our own caller
-              // can't use passthrough() here because that demands a JSON response
-              externalHttpPerRequest(requestContext,
-                Post(calhounStaticNotebooksURL, requestContext.request.entity))
-          }
+          requestContext =>
+            // call Calhoun and pass its response back to our own caller
+            // can't use passthrough() here because that demands a JSON response
+
+            //TODO: ensure mediatype is still honored, but respondWithMediaType was deprecated
+            //because it was an anti-pattern
+
+            val extReq = Post(calhounStaticNotebooksURL, requestContext.request.entity)
+            executeRequestRaw(userInfo.accessToken)(extReq).flatMap { resp =>
+              requestContext.complete(resp)
```

Comment:
i need to actually test that this returns `text/html` still. if not i'll have to make sure the content type negotiation is working, because `respondWithMediaType` was removed in akka-http since it was an anti-pattern

Reply:
I tested this empirically, and see a response header of `content-type: text/html; charset=UTF-8 `